doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,201 @@
+
+.. _zigbee_commissioning_modes_nf:
+
+Network Formation
+=================
+
+The Network Formation mode is available only to Coordinators and Routers, because End Devices are not capable of forming a Zigbee network.
+
+.. figure:: /images/zigbee_commissioning_formation.svg
+   :alt: Zigbee Network Formation procedure
+
+   Zigbee Network Formation procedure
+
+In this mode, the node has not found a network to join during the Network Steering mode, and so it attempts to form a new network.
+Depending on the node role:
+
+* A Coordinator forms a centralized security network.
+* A Router forms a distributed security network.
+
+If Network Formation is successful, you can add nodes to the new network using `Network Steering`_.
+If unsuccessful, the commissioning procedure stops here.
+
+Network Formation in the |NCS|
+   In the ZBOSS stack, Network Formation is started using the `ZB_BDB_NETWORK_FORMATION`_ bit mask.
+
+   In the |NCS|, when using the :ref:`lib_zigbee_signal_handler` and the application implements the Zigbee Coordinator role, the Network Formation procedure is started right after the stack initialization.
+
+Centralized security network
+----------------------------
+
+The centralized security network is started by the Zigbee Coordinator node, which assumes the Trust Center functionality.
+As such, it is responsible for managing the security and authorization in the network.
+In the centralized security network, only the Trust Center can provide the network key to the joining device.
+This happens by using a link key specific to the joining device.
+The link key is exchanged with the Trust Center, which allows for encrypting packets exchanged between the Trust Center and the new node.
+This way, only the Trust Center and the new node can decrypt the packages exchanged between them.
+The link key value can be defined in one of the following ways:
+
+* Calculated based on the joining device's install code.
+* Predefined in the specification: ``"ZigBeeAlliance09"``.
+
+Distributed security network
+----------------------------
+
+The distributed security network is formed by Routers and does not have a centralized node that manages security in the network.
+In this network type, all Routers are equal and there is no root of trust between them.
+Each Router can provide the network key to the joining device.
+This happens by using a distributed security global link key.
+The link key value is provided as a result of a successful product certification.
+
+.. _zigbee_commissioning_modes_fb:
+
+Finding and Binding
+===================
+
+After the node joins a network, it can undergo the Finding and Binding procedure.
+With this procedure, you can establish a connection between corresponding endpoints on two different nodes automatically.
+As a result, new binding table entries are created and the nodes do not have to use the device address to communicate, but can rely on connections between the same clusters in their binding tables.
```

Comment:
When it comes to binding, it's the endpoint at the node device that is connected to an endpoint at another node device.
Should we be more specific here?

Reply:
What about:
`As a result, new binding table entries are created and the nodes do not have to use the device addresses to communicate, but can rely on connections between the same endpoints in their respective cluster binding tables.`?

Reply:
Looks fine for me